docs/src/integrations/source/troubleshooting.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Troubleshooting"
+description: |
+  In the following section you'll find troubleshooting guidance on source integration.
+---
+
+{{< description >}}
+
+## Troubleshooting while cloning the repository
+
+When you set up an [external integration](https://docs.platform.sh/integrations/source.html) to GitHub, GitLab, or Bitbucket, an additional layer of access control gets added.
+
+### Example
```

Comment:
This doesn't seem like a necessary level of structure. This whole intro should be about what kind of error people might encounter.